tests/access_test.go

```diff
@@ -241,4 +245,80 @@ var _ = Describe("[rfe_id:500][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			table.Entry("on vm restart", "virtualmachines", "restart"),
 		)
 	})
+
+	Describe("With regular OpenShift user", func() {
+		BeforeEach(func() {
+			tests.SkipIfNoCmd("oc")
+		})
+
+		const testUser = "testuser"
+
+		testRights := func(resource, right string) {
+			verbsList := []string{"get", "list", "watch", "delete", "create", "update", "patch", "deletecollection"}
+
+			for _, verb := range verbsList {
+				// AS A TEST USER
+				By(fmt.Sprintf("verifying user rights for verb %s", verb))
+				result, _, _ := tests.RunCommand(k8sClient, "auth", "can-i", "--as", testUser, verb, resource)
+				Expect(result).To(ContainSubstring(right))
+			}
+		}
+
+		Context("should fail without admin rights for the project", func() {
+			BeforeEach(func() {
+				By("Ensuring the cluster has new test user")
+				stdOut, stdErr, err := tests.RunCommandWithNS("", k8sClient, "create", "user", testUser)
```

Comment:
what does "" stand for?

Reply:
This command runs a command without `--namespace kubevirt-test-default`. The command used elsewhere `tests.RunCommand()` auto add the mentioned. 

I specifically require to run add user without namespace, since this is the use case user will do.